src/components/Buttons.js

```diff
@@ -1,32 +1,32 @@
-/* eslint-disable react/destructuring-assignment */
-/* eslint-disable react/prop-types */
-/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import PropTypes from 'prop-types';
 
-export default class Buttons extends React.Component {
-  render() {
-    return (
-      <div className="btn-subgrid">
-        <button type="button" onClick={() => this.props.clickHandler('AC')}>AC</button>
-        <button type="button" onClick={() => this.props.clickHandler('+/-')}>+/-</button>
-        <button type="button" onClick={() => this.props.clickHandler('%')}>%</button>
-        <button type="button" className="orange" onClick={() => this.props.clickHandler('÷')}>÷</button>
-        <button type="button" onClick={() => this.props.clickHandler('7')}>7</button>
-        <button type="button" onClick={() => this.props.clickHandler('8')}>8</button>
-        <button type="button" onClick={() => this.props.clickHandler('9')}>9</button>
-        <button type="button" className="orange" onClick={() => this.props.clickHandler('x')}>x</button>
-        <button type="button" onClick={() => this.props.clickHandler('4')}>4</button>
-        <button type="button" onClick={() => this.props.clickHandler('5')}>5</button>
-        <button type="button" onClick={() => this.props.clickHandler('6')}>6</button>
-        <button type="button" className="orange" onClick={() => this.props.clickHandler('-')}>-</button>
-        <button type="button" onClick={() => this.props.clickHandler('1')}>1</button>
-        <button type="button" onClick={() => this.props.clickHandler('2')}>2</button>
-        <button type="button" onClick={() => this.props.clickHandler('3')}>3</button>
-        <button type="button" className="orange" onClick={() => this.props.clickHandler('+')}>+</button>
-        <button type="button" className="span_two" onClick={() => this.props.clickHandler('0')}>0</button>
-        <button type="button" onClick={() => this.props.clickHandler('.')}>.</button>
-        <button type="button" className="orange" onClick={() => this.props.clickHandler('=')}>=</button>
-      </div>
-    );
-  }
+export default function Buttons(props) {
+  Buttons.propTypes = {
```

Comment:
- [OPTIONAL] Following ES6 syntax, you could implement it to your other components too. ♻️

Reply:
Thanks a lot, @ArthurGC  I'll definitely do so :smiley:  :nerd_face:

Reply:
Actioned :heavy_check_mark: